src/response/chi0.jl

```diff
@@ -291,49 +292,60 @@ Perform in-place computations of the derivatives of the wave functions by solvin
 a Sternheimer equation for each `k`-points. It is assumed the passed `δψ` are initialised
 to zero.
 """
-function compute_δψ!(δψ, basis, H, ψ, εF, ε, δHψ; ψ_extra=[zeros_like(ψk, size(ψk,1), 0) for ψk in ψ],
-                     kwargs_sternheimer...)
+# For phonon calculations, the equation to be solved is
+#   (H_k - ε_{n,k-q}) δψ_{n,k} = - Q δHψ_{n, k-q},
+# as the phonon modes couple the k and k-q points.
+# Furthermore, for temperature computations, we also need the eigenvalues at the k point.
```

Comment:
doesn't really matter why it's needed

Reply:
just document what they are: eps_p is at k-q, eps is at k. Also weird names, eps_k and eps_k_minus_q ?

Reply:
This is super confusing, maybe let's discuss how to document this